tsconfig.json

```diff
@@ -1,11 +1,21 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
-    "rootDir": "typescript",
+    "rootDir": ".",
     "outDir": "build",
-    "baseUrl": "typescript"
+    "baseUrl": ".",
+    "allowJs": true,
+    "lib": [
+      "es2016"
```

Comment:
Any reason to not just let `gts` pick `es2018`?

Reply:
nope! fixed